clarity/contracts/arkadiko-stake-registry-v1-1.clar

```diff
@@ -117,15 +117,37 @@
 
 ;; Initialize the contract
 (begin
-  ;; Add initial contracts
+  ;; DIKO pool
   (map-set pools-data-map
     { pool: 'STSTW15D618BSZQB85R058DS46THH86YQQY6XCB7.arkadiko-stake-pool-diko-v1-1 }
     {
       name: "Diko",
       active: true,
       activated-block: block-height,
       deactivated-block: u0,
-      rewards-percentage: u1000000 ;; 100% 
+      rewards-percentage: u200000 ;; 20% 
+    }
+  )
+  ;; DIKO-xUSD LP
+  (map-set pools-data-map
+    { pool: 'STSTW15D618BSZQB85R058DS46THH86YQQY6XCB7.arkadiko-stake-pool-diko-xusd-v1-1 }
+    {
+      name: "DIKO-xUSD LP",
+      active: true,
+      activated-block: block-height,
+      deactivated-block: u0,
+      rewards-percentage: u400000 ;; 40% 
```

Comment:
Q: How did you decide 40% for each?

Are we more interested in liquidity on DIKO/xUSD rather than wSTX/xUSD? Maybe put the needle slightly to DIKO with a 20 / 50 / 30? or 20 / 45 / 35?

Reply:
I could not decide ;) 

I'm a bit worried that the rewards to provide wSTX/xUSD are not high enough. Seems much better to just put STX in a vault. You get pox rewards, DIKO rewards and can mint xUSD. With the DIKO rewards and minted xUSD you can start providing liquidity. You again earn DIKO which can be used to add to your LP position. In this scenario you keep full STX exposure while earning pox rewards, so the end result for an STX holder is always positive. 

But if you want to provide wSTX/xUSD liquidity you need to put some STX in the pool directly. You will suffer impermanent loss and won't be able to stack. In this scenario an STX holder could end up negative compared to just stacking STX, so this is a higher risk. This pool might be important for xUSD as it allows arbitrage so helps stabilising xUSD value?

Reply:
@nieldeckx good points - didn't think of that. Maybe we should do it the other way around then even, more rewards for wSTX/xUSD :)

20 / 30 / 50 or 20 / 35 / 45. I'll CC in ozzy through chat